### ISSUE TITLE
false-positive for newspaper-order when using getter/setter with helper function

### ISSUE THREAD

**Issue by stenzengel** [Glavin001/tslint-clean-code#51]
When linting the following test class with enabled rule "newspaper-order" a tslint warning is shown. That looks like a false positive to me.

```typescript
class Test {
  private _value: number;

  get value(): number {
    return this._value;
  }

  set value(value: number) {
    // If you comment out the following line, no warning occurs.
    if (this.isPossibleValue(value))
      this._value = value;
  }

  isPossibleValue(value: number): boolean {
    return value !== this._value;
  }
}
```

Warning:
```
The class does not read like a Newspaper. Reorder the methods of the class: Test

Methods order:
1. ✓ value
2. x isPossibleValue
```

If I do not call "isPossibleValue" in the setter. No warning is shown. This is correct behaviour.

**Comment by Glavin001** [comment]
Thanks for reporting! It would be great if you could make a Pull Request adding your failing test case to https://github.com/Glavin001/tslint-clean-code/blob/master/src/newspaperOrderRule.ts . If you are able to fix it, too, even better! Let me know if you need any assistance.

**Comment by stenzengel** [comment]
Closing, because tslint is now deprecated.
